src/azure-cli/azure/cli/command_modules/rdbms/commands.py

```diff
@@ -458,3 +483,21 @@ def load_command_table(self, _):
         g.command('delete', 'delete', confirmation=True)
         g.show_command('show', 'get')
         g.wait_command('wait')
+
+    with self.command_group('mysql server',
+                            mysql_location_based_performance_tier_sdk,
+                            client_factory=cf_mysql_location_based_performance_tier_operations,
+                            is_preview=True) as g:
+        g.command('list-skus', 'list', table_transformer=table_transform_output_list_skus_single_server)
+
+    with self.command_group('postgres server',
+                            postgres_location_based_performance_tier_sdk,
+                            client_factory=cf_postgres_location_based_performance_tier_operations,
+                            is_preview=True) as g:
+        g.command('list-skus', 'list', table_transformer=table_transform_output_list_skus_single_server)
+
+    with self.command_group('mariadb server',
+                            mariadb_location_based_performance_tier_sdk,
+                            client_factory=cf_mariadb_location_based_performance_tier_operations,
+                            is_preview=True) as g:
+        g.command('list-skus', 'list', table_transformer=table_transform_output_list_skus_single_server)
```

Comment:
please add tests for all these new commands.

Reply:
Tests including these new commands are added in this pull request https://github.com/Azure/azure-cli/pull/15363

Reply:
Will do